test/benchmark/shell_scripts/iGenVar_init.sh

```diff
@@ -0,0 +1,51 @@
+# -------- -------- Prepare logfile and runtime computation -------- -------- #
+mkdir logs
+echo "Logifle written to: initial_steps.log"
+exec 3>&1 1>>logs/initial_steps.log 2>&1                                # write stdout 1,2 to logfile 3 to console
+echo $(date)
+start=$(date +%s)                                                       # get starting date
```

Comment:
```suggestion
#!/usr/bin/env sh
echo "Logfile written to: initial_steps.log"

# -------- -------- Prepare logfile and runtime computation -------- -------- #
mkdir -p logs

exec 3>&1 4>&2 # store original streams
trap 'exec 2>&4 1>&3' 0 1 2 3 # restore original streams when scripts ends
exec 1>logs/initial_steps.log 2>&1 # write stdout and stdcerr to logfile

set -ex

echo $(date)
start=$(date +%s)                                                       # get starting date
```

* `#!/usr/bin/env sh` scripts should have a shebang
* `mkdir -p` will not complain if the directory exists, without `-p` there will be an error
* the `exec`/`trap` stuff is just a way I found to do this quite reliably, but the effect of writing everything to a logfile is the same 
* I did `1>logs/initial_steps.log` instead of `1>>logs/initial_steps.log` since you usually don't want to append if you run the file multiple times. But if this is on purpose, use `>>` instead :)
* `set -ex`: `-e` will stop at an error, this is usually a good idea. `-x` means echoing the commands when executed. This makes it easier to track what's happening, but it's of course more verbose. I'd recommend `-e`, `-x` depends on your taste :)


This also applies to the other scripts